.github/actions/bump-version/action.yml

```diff
@@ -0,0 +1,42 @@
+name: Bump Version
+
+description: Version affected packages
+
+inputs:
+  github_token:
+    description: GitHub token
+    required: true
+  npm_token:
+    description: NPM token
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: install dependencies
+      env:
+        NODE_AUTH_TOKEN: ${{ inputs.npm_token }}
+      shell: bash
+      run: npm ci --legacy-peer-deps
+
+    - name: Setup git user to "GSA Bot"
+      shell: bash
+      run: git config user.email "-" && git config user.name "🤖 GSA Bot"
+
+    - name: Version
+      env:
+        NODE_AUTH_TOKEN: ${{ inputs.npm_token }}
+        GITHUB_TOKEN: ${{ inputs.github_token }}
+      shell: bash
+      run: npx nx affected --base=last-release --parallel=1 --target=version
+
+    - name: Push changes
+      shell: bash
+      run: git push origin HEAD
+    - name: Push tags
+      shell: bash
+      run: git push --tags
+    - name: Tag last-release
+      shell: bash
+      run: |
+          git tag -f last-release
```

Comment:
I think the `last-release` tag should be different for a specific library. Can we use a library release tag instead, e.g. `lib1@vX.Y.Z`?

Reply:
Here this last-release tag is used for computing the affected packages ` npx nx affected --base=last-release --parallel=1 --target=version`
And this last-release tag is always pushed whenever the CI workflow will be triggered for publishing any package.